textacy/compat.py

```diff
@@ -3,9 +3,13 @@
 PY2 = int(sys.version[0]) == 2
 
 if PY2:
+    import bz2
```

Comment:
Nope. I was just following the same convention as for the other Python version-specific imports, but this module is the same in both — will fix.